project1/server/agent.cpp

```diff
@@ -180,15 +182,18 @@ namespace server {
         }
 
         // monitor for termination request for get and put commands
-        uint16_t kBytesSent = 0;
-        const uint32_t kBytesToSend = 1000;
+        uint16_t total_bytes_sent = 0;
```

Comment:
Probably want this not to be a `uint16_t`. If they're sending 5 MB files, this is going to overflow.